src/logging.cc

```diff
@@ -2627,6 +2654,10 @@ void EnableLogCleaner(const std::chrono::minutes& overdue) {
   log_cleaner.Enable(overdue);
 }
 
+void EnableLogCleaner(std::unordered_map<LogSeverity, std::chrono::minutes>& overdue_per_severity) {
+  log_cleaner.Enable(overdue_per_severity);
+}
+
 void DisableLogCleaner() { log_cleaner.Disable(); }
```

Comment:
For symmetry, there should be an API call for disabling the log cleaner for specific severity.